src/main/g8/src/main/scala/$package$/Main.scala

```diff
@@ -1,18 +1,15 @@
 package $package$
 
-import java.io.IOException
-
-import zio.{  ZIOAppDefault, ExitCode, URIO, ZEnv, ZIO, Console }
-import zio.Console.{ printLine, readLine }
+import zio.{ZIOAppDefault, Task, ZIO, Console}
+import zio.Console.{ConsoleLive, printLine, readLine}
 
 object Main extends ZIOAppDefault:
 
-  val app: ZIO[Console, IOException, Unit] =
+  def app(c: Console): Task[Unit] =
     for
-      _    <- printLine("What is your name?")
+      _ <- printLine("What is your name?")
```

Comment:
Shouldnt it be `c.printLine`? @lemastero

Reply:
Good spot! Fixed https://github.com/ScalaConsultants/zio-dotty-quickstart.g8/pull/96/commits/90636ad06d955b20412ece9d5d045305555f1292